src/ontology/mondo-edit.obo

```diff
@@ -508346,17 +508346,21 @@ is_a: MONDO:0003847 {source="OMIM:618598"} ! hereditary disease
 [Term]
 id: MONDO:0032829
 name: neurodevelopmental disorder with hypotonia and variable intellectual and behavioral abnormalities
+def: "Neurodevelopmental disorder in which the cause of the disease is a variation in the POLR2A gene; it is characterized by early-onset hypotonia, delayed walking, poor speech, and impaired intellectual development. Other features may include feeding difficulties, dysmorphic features, and visual problems. Brain imaging tends to show delayed myelination, thin corpus callosum, and/or enlarged ventricles." [https://orcid.org/0000-0001-9310-0163, PMID:31353023, PMID:35328024]
```

Comment:
@sabrinatoro Would it be okay to change "brain imaging" to "brain magnetic resonance imaging"? Just to be more precise about the modality of imaging? I am happy to change this if you agree.

Reply:
@katiermullen "brain imaging" is more general, so I would be tempted to keep it in case other brain imaging that is not brain magnetic resonance imaging, showed the same thing. 
That being said, I don't know anything about brain imaging. So I am asking you, the expert :-) : is brain magnetic resonance imaging the (only) way to see these symptoms in patients? If so, it makes total sense to change.

Reply:
yes, it is the only type of imaging that would show these lesions. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6699192/ specifically mentions brain MRI abnormalities in Table 1. I will update the definition and merge the PR. Thank you!